src/migrations/20240603092044-payment-momo.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-undef */
+'use strict';
+
```

Comment:
You mean you can't add another filed in a model without clearing the first migration? @Sagastein

Reply:
it can work but the migrations cant be applied to the database unless i delete it in sequelizeMeta table.
without generating a new migrations when you try to apply the migration to the table you will always get that the database is already updated.